Card/task 1

gkartsivadze opened the pull request:
Component gets product as object : name, price, discount, image, favourite (boolean), cart (boolean).
In case of discount, shows old price in red and discounted one in black.
to prevent product name from exceeding component borders, it's length is limited on 27 symbols.

!Component includes default product for testing!

gkartsivadze committed:
ავაგე HTML ჩონჩხი, დავამატე სატესტო პროდუქტი.

gkartsivadze committed:
კომპონენტის გასტილვა
დავამატე სატესტო პროდუქტის ობიექტს ფავორიტების და კალათის ბულეანი

gkartsivadze committed:
პროდუქტის სახელის სიგრძის ლიმიტირება 27 სიმბოლოზე

MerabTskhvedadze commented on src/Components/Card/Card.jsx:
images-foldershi card folderi sheqmeni da cardshi ra fotoebsac iyeneb iq sheitane egeni, svg-ebi rogaq egeni komponentad daamate images foldershi damatebul card-foldershi da iqidan daaimporte da eg komponentebi gamoiyene button-ebshi

gkartsivadze replied:
ჩასწორებულია

gkartsivadze committed:
ჩასწორება
აიქონების ცალკე ფაილად გამოტანა
ფოტოების ცალკე ფოლდერში გადატანა

gkartsivadze committed:
updated Card.jsx

gkartsivadze committed:
svg ფაილები ჩავასწორე კომპონენტებით

MerabTskhvedadze reviewed (commented):
good job!

gkartsivadze committed:
ჩასწორება :
პროდუქტის ფავორიტებში ან კალათში არსებობის შემთხვევაში აიქონები ფერს არ იცვლიდა.
